cmd/buildiso.go

```diff
@@ -15,7 +17,63 @@ var buildisoCmd = &cobra.Command{
 	Long:  "Build all profiles into a bootable CD image. All flags are optional.",
 	Run: func(cmd *cobra.Command, args []string) {
 		generateCobblerClient()
-		// TODO: call cobblerclient
+		isoOption, err := cmd.Flags().GetString("iso")
+		if err != nil {
+			fmt.Println(err)
```

Comment:
Instead of using `Run:` you can use `RunE`:

```go
		RunE: func(cmd *cobra.Command, args []string) error {
			generateCobblerClient()

			isoOption, err := cmd.Flags().GetString("iso")
			if err != nil {
				return err
			}

			// ...

			return nil
		},
```